test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlWriterTests.cs

```diff
@@ -507,6 +507,96 @@ public void ShouldWriteOptionalParameters()
 
         #endregion
 
+        [Fact]
+        public void ShouldWriteEdmComplexTypeProperty()
+        {
+            string expected =
+            "<?xml version=\"1.0\" encoding=\"utf-16\"?>" +
+            "<edmx:Edmx Version=\"4.0\" xmlns:edmx=\"http://docs.oasis-open.org/odata/ns/edmx\">" +
+              "<edmx:DataServices>" +
+                "<Schema Namespace=\"NS\" xmlns=\"http://docs.oasis-open.org/odata/ns/edm\">" +
+                  "<EntityType Name=\"Customer\">" +
+                    "<Key>" +
+                      "<PropertyRef Name=\"Id\" />" +
+                    "</Key>" +
+                    "<Property Name=\"Id\" Type=\"Edm.Int32\" Nullable=\"false\" />" +
+                    "<Property Name=\"Properties\" Type=\"Edm.ComplexType\" />" +

```

Comment:
>Properties [](start = 38, length = 10)

Nit; should probably be "Property", not "Properties", as it is a single complex value.  Maybe even name it "ComplexProperty".

Reply:
#Resolved.